src/Zone.js

```diff
@@ -35,17 +34,31 @@ export default class Zone extends UXBase {
 		return mark;
 	});
 
+	// Parent View object this zone belongs to
+	view = this.props.view;
+
 	/**
 	 * Records measure on Performance Timeline and calls onMeasure callback
 	 *
 	 * @param {ExpectedMark} lastMark last mark that triggered completion
 	 */
 	measure(endMarkName) {
+		if (!this.view.startMarkName) {
+			throw new Error(
+				`[UX Capture] Call to measure a zone in the view before view was started using startTransition() method.
```

Comment:
If view is a `PageView` there is no `startTransition()` method. Not sure I understand what this error means.

Reply:
Basically, it means that startMarkName is only available in either PageView mode or in InteractiveView mode, but only after startTransition() is called. If it's called before, we don't know where to start the measure.

Will referring to interactive view in the error message be enough to clarify this?

Reply:
I'd just go with something simple that spells out the error rather than try to explain a use case or implementation detail that might change later. i.e., maybe something like

`[UX Capture Error] Missing view.startMarkName in Zone.measure()`